src/main/java/com/synopsys/integration/detect/workflow/componentlocationanalysis/ScanResultToComponentListTransformer.java

```diff
@@ -25,33 +27,40 @@ public class ScanResultToComponentListTransformer {
      * @param rapidScanFullResults
      * @return list of {@link Component}s
      */
-    public List<Component> transformScanResultToComponentList(List<DeveloperScansScanView> rapidScanFullResults) {
+    public Set<Component> transformScanResultToComponentList(List<DeveloperScansScanView> rapidScanFullResults) {
```

Comment:
comment and method name could use reworking to reflect this is a set

Reply:
Done